classes/Tools.php

```diff
@@ -2509,9 +2509,10 @@ public static function generateHtaccess($path = null, $rewrite_settings = null,
         fwrite($write_fd, "AddType application/vnd.ms-fontobject .eot\n");
         fwrite($write_fd, "AddType font/ttf .ttf\n");
         fwrite($write_fd, "AddType font/otf .otf\n");
+        fwrite($write_fd, "AddType font/woff2 .woff2\n");
```

Comment:
If we refer to the w3c spec, it should be font/woff2

source: https://dev.w3.org/webfonts/WOFF2/spec/#IMT